rose/common/config.py

```diff
@@ -9,13 +9,7 @@
 
 game_rate = 1.0
 game_duration = 60
-
-# Client
-
-frame_rate = 30
 number_of_cars = 4
-car_jitter = 10
-play_sound = True
```

Comment:
The "Dashboard", "Finish line", and "Window" are also not needed now.

The the "Files" section, we can remove the paths to the various resources (e.g. obstables_dir). I think the server is using only the install_dir to serve the resources.

Reply:
> The the "Files" section, we can remove the paths to the various resources (e.g. obstables_dir). I think the server is using only the install_dir to serve the resources.

Yup, just tried that. Only keeping the `install_dir` (well, and `web_root` and `res_root`) seems to work. I wasn't sure about removing that first so I kept it in. Thanks!